src/types.ts

```diff
@@ -33,14 +33,14 @@ export enum OperatingSystems {
 
 export enum Services {
 	appStore = 'appStore',
-	detection = 'detection',
-	integration = 'integration',
-	notificationRule = 'notificationRule',
+	detection = 'detections',
+	integration = 'integrations',
+	notificationRule = 'notificationRules',
```

Comment:
@jasper-dassana I will update service map as well so this should become
```suggestion
	detections = 'detections',
	integrations = 'integrations',
	notificationRules = 'notificationRules',
```